scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname VARCHAR,
+  given_name VARCHAR,
+  gender VARCHAR(1),
+  born_on DATE,
+  specialty VARCHAR
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname VARCHAR,
+  given_name VARCHAR,
+  gender VARCHAR(1),
+  born_on DATE,
+  insurance VARCHAR
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date TEXT,
```

Comment:
The prompt said feel free to use TEXT for all of the fields, so I did.

"appointments - which should have date, time, and comment columns (feel free to use the type TEXT for all three) as well as columns referencing both doctors and patients."